### ISSUE TITLE
`include` isn't completely exclusive

### ISSUE THREAD

**Issue by plocket** [BryceStevenWilley/DAYamlStructure#1]
https://github.com/BryceStevenWilley/DAYamlStructure/blob/526be711c5682c90b22e94793574e1402957ee62/da_yml_structure.py#L204-L205

I've tested and it can have `comment` and `id` at the very least. We could decide that it shouldn't, but those are things I'd probably find useful (maybe have already) in an `include` block.

**Comment by BryceStevenWilley** [comment]
I agree with comment, TBH I should treat that one separately somehow, since it should be able to go anywhere, including multiple times in a block IMO.

I'm more skeptical of id. You're right that it can go on anything, but from what I can tell, the only thing DA lets you do with it on non-question blocks is [some wacky manual re-ordering](https://docassemble.org/docs/initial.html#order). From a maintenance / engineering perspective, it's a horrible idea and definitely one of those things I'd like to be strict about from a validator / linter perspective.

**Comment by plocket** [comment]
Makes sense. Just fyi, `id` can't go on everything. Not sure when it happened, but it gave me an error one time. I _think_ it was when the only other property was a `comment`.

**Comment by nonprofittechy** [comment]
We don't actually enforce this limit at the moment. But see #57 for a better place to deal with this. will require reviewing tests + boundaries that aren't checked yet
